MediaBrowser.Model/Dto/DisplayPreferencesDto.cs

```diff
@@ -63,10 +63,10 @@ public DisplayPreferencesDto()
         public int PrimaryImageWidth { get; set; }
 
         /// <summary>
-        /// Gets or sets the custom prefs.
+        /// Gets the custom prefs.
         /// </summary>
         /// <value>The custom prefs.</value>
-        public Dictionary<string, string?> CustomPrefs { get; set; }
+        public Dictionary<string, string?> CustomPrefs { get; }
```

Comment:
I think this will break setting display preferences

Reply:
You're right